vars/cloudFoundryDeploy.groovy

```diff
@@ -222,6 +227,15 @@ void call(Map parameters = [:]) {
             .withMandatoryProperty('cloudFoundry/credentialsId')
             .use()
 
+        if (config.useGoStep == true) {
```

Comment:
Minor:
```suggestion
        if (config.useGoStep) {
```

Reply:
Consider the following:

```yaml
steps:
  cloudFoundryDeploy:
    useGoStep: 'false'
```

`useGoStep` is now a String as far as the YAML parser is concerned and the Groovy Truth would result in something unintended. I've written a small test and it will indeed behave differently:

```groovy
@Test
void groovyTruthForStrings() {
    Map config = [
        someString: 'false'
    ]

    if (config.someString == true) {
        assertTrue(false)
    } else {
        assertTrue(true)
    }

    if (config.someString) {
        assertTrue(false) // The test will fail here!
    } else {
        assertTrue(true)
    }
}
```